include/wx/webview.h

```diff
@@ -191,7 +191,8 @@ class WXDLLIMPEXP_WEBVIEW wxWebView : public wxControl
     virtual wxString GetUserAgent() const;
 
     // Script
-    virtual bool RunScript(const wxString& javascript, wxString* output = NULL) const = 0;
+    virtual bool RunScript(const wxString& javascript, wxString* output = NULL) const;
```

Comment:
Do we need to document the parameter type change in "Incompatible changes" section or was this method never supposed to be overridden in the application code and so it doesn't matter?

Reply:
I think this shouldn't matter. Only other backend implementations (of which there are probably VERY few) would override this. And even if they do (like the IE backend) they still can.